README.md

```diff
@@ -28,12 +27,12 @@ Simply add the following line to your ```composer.json``` and run ```composer up
 ```
 Or use composer to add it with the following command
 ```bash
-composer require "hihaho/laravel-encryptable-trait:^v1.3"
+composer require hihaho/laravel-encryptable-trait
 ```
 
 ## Requirements
-- illuminate/encryption ^6.0, ^7.0 or ^8.0
-- PHP 7.2, 7.3, 7.4 or 8.0
+- illuminate/encryption ^8.75 or ^9.0
```

Comment:
```suggestion
- illuminate/encryption ^8.0 or ^9.0
```

Reply:
PHP8.1 does not work for earlier versions of `illuminate/encryption` unfortunately, tested this by increasing the version step by step and running `composer update --prefer-lowest` until the tests passed